src/20ec0_len_5040.c

```diff
@@ -1,5 +1,60 @@
 #include "common.h"
 #include "npc.h"
+#include "hud_element.h"
+
+typedef struct HudComplexStatusIcon {
+	/* 0x0 */ s8 active;
+	/* 0x1 */ s8 removing;
+	/* 0x2 */ s8 unk_02;
+	/* 0x3 */ s8 unk_03;
+	/* 0x4 */ s8 unk_04;
+	/* 0x5 */ s8 unk_05;
+	/* 0x8 */ s32 hudElemIndex;
+	/* 0xC */ s32 unk_0C;
+} HudComplexStatusIcon;/* size = 0x10 */
+
+typedef struct HudSimpleStatusIcon {
+	/* 0x0 */ s8 active;
+	/* 0x1 */ s8 removing;
+	/* 0x4 */ s32 hudElemIndex;
+} HudSimpleStatusIcon;/* size = 0x8 */
+
+typedef struct HudStatusIcon {
+	/* 0x000 */ s32 flags;
+	/* 0x004 */ Vec3f worldPos;
+	/* 0x010 */ f32 status1Radius;
+	/* 0x014 */ f32 status1OffsetY;
+	/* 0x018 */ f32 status2Radius;
+	/* 0x01C */ f32 status2OffsetY;
+	/* 0x020 */ f32 status3Radius;
+	/* 0x024 */ f32 status3OffsetY;
+	/* 0x028 */ f32 status4Radius;
+	/* 0x02C */ f32 status4OffsetY;
+	/* 0x030 */ f32 offsetY;
+	/* 0x034 */ f32 unk_34;
+	/* 0x038 */ HudComplexStatusIcon status1;
+	/* 0x048 */ HudComplexStatusIcon status2;
+	/* 0x058 */ HudComplexStatusIcon status3;
+	/* 0x068 */ HudComplexStatusIcon status4;
+	/* 0x078 */ HudSimpleStatusIcon boostJump;
+	/* 0x080 */ s32 prevIndexBoostJump;
+	/* 0x084 */ HudSimpleStatusIcon boostHammer;
+	/* 0x08C */ s32 prevIndexBoostHammer;
+	/* 0x090 */ HudSimpleStatusIcon boostPartner;
+	/* 0x098 */ HudSimpleStatusIcon surprise;
+	/* 0x0A0 */ HudSimpleStatusIcon peril;
+	/* 0x0A8 */ HudSimpleStatusIcon danger;
+} HudStatusIcon;/* size = 0xB0 */
```

Comment:
I'd prefer to use the normal ` // size = ` comment format like alex said above. also not to be too pedantic, but since this only goes up to 2 hex digits, the leading 0s can be removed from the offset comments in this struct as well